web/js/modules/location-search/util.js

```diff
@@ -154,18 +154,20 @@ export function serializeCoordinatesWrapper(coordinates, state) {
   const { map, proj } = state;
   const serializeCoordinates = (coordinate) => {
     const coordinateValues = [coordinate.latitude, coordinate.longitude];
-    if (map.ui.selected) {
-      const coordinatesWithinExtent = areCoordinatesWithinExtent(proj, coordinateValues);
-      if (!coordinatesWithinExtent) {
-        return;
-      }
-    }
+    if (!map.ui.selected) return null;
```

Comment:
Not a big deal but you shouldn't need to return `null` unless whatever is calling this is looking for that value explicitly. `null`/`undefined`/`0`/`false` are all "falsy" and default `return` returns `undefined` so I would guess you're fine just returning.